tests/orquestra/integrations/cirq/simulator/qsimsimulator_test.py

```diff
@@ -93,6 +93,19 @@ def test_get_wavefunction(self):
             wavefunction.amplitudes[7], (1 / np.sqrt(2) + 0j), atol=10e-15
         )
 
+    def test_normalization(self):
```

Comment:
I'm not sure what this test is actually testing.
It looks to me that it checks whether the normalized wavefunction is "closer to 1" than the unnormalized one.
But it took me a moment to parse, so what about the following refactor:

```
precision_error_without_normalization = np.abs(np.sum(np.abs(wavefunction.amplitudes) ** 2) - 1.0)
precision_error_with_normalization = np.abs(np.sum(np.abs(normalized_wavefunction.amplitudes) ** 2) - 1.0)
assert not np.isclose(error_without_normalization, 0)
assert error_with_normalization == 0
```
Or something along these lines.
This way we also make sure that without normalization we actually get into precision issues for this circuit.

Reply:
This has been dealt with 😄